### PR TITLE
docs(workspace): add GitHub templates and update samples docs

### PR DESCRIPTION
**Pull request by jorgeramirezamora**
## Changes
- Add Next.js sample application using `presto-js-client`.
- Add issue_templates and pull_request_template.
- Update sample applications README files (Next.js and NestJS).

## Notes

N/A

## Screenshots

image

### CODE REVIEW

**Comment by jorgeramirezamora** [comment]
For some reason I cannot add @yhwang as reviewer. Can u plz add him @tdcmeehan?

**Comment by tdcmeehan** [comment]
> For some reason I cannot add @yhwang as reviewer. Can u plz add him @tdcmeehan?

Added, and I have also added @yhwang as a contributor to this project, so you should be able to do this in the future.

File: apps/nest-server/README.md [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+2. Once the Presto instance is up and running, you can start the Nest Server by running:
+
+   ```bash
+   npm run serve nest-server
+   ```
+
+3. After starting the Nest Server, you can make a `GET` request to the following URL to see the response from querying the local Presto container using the `prestodb-js-client` library:
+
+   ```
+   http://localhost:3000/api/call-centers
```
Comment (yhwang) [inline apps/nest-server/README.md]: do you want to have a curl command here or just say open a browser with this URL? That would be more specific. And maybe add a section with the sample outputs.

Comment (yhwang) [reply apps/nest-server/README.md]: Like the step 3 you have in the nextjs's README.

File: apps/nextjs/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+   ```bash
+   npm run serve nextjs
+   ```
+
+3. Once the Next.js application is running, open your web browser and navigate to the following URL:
+
+   ```
+   http://localhost:4200/
+   ```
+
```
Comment (yhwang) [inline apps/nextjs/README.md:26]: Having sample outputs/screenshots would be better.

File: apps/nest-server/README.md (hunk 1 above)
Comment (jorgeramirezamora) [reply apps/nest-server/README.md]: A curl command sounds good 👌🏼 Will add sample outputs as well

File: apps/nextjs/README.md (hunk 2 above)
Comment (jorgeramirezamora) [reply apps/nextjs/README.md:26]: Will add sample outputs as well

Comment (jorgeramirezamora) [reply apps/nextjs/README.md:26]: Done!

File: apps/nest-server/README.md (hunk 1 above)
Comment (jorgeramirezamora) [reply apps/nest-server/README.md]: Done!

File: apps/nest-server/README.md [hunk 3]
```diff
@@ -1,9 +1,31 @@
...
+2. Once the Presto instance is up and running, you can start the Nest Server by running:
+
+   ```bash
+   npm run serve nest-server
+   ```
+
+3. After starting the Nest Server, you can make a `GET` request to the following endpoint to see the response from querying the local Presto container using the `prestodb-js-client` library:
+
+   ```
+   curl -i -H "Accept: application/json" -H "Content-Type: application/json" -X GET http://localhost:3000/api/call-centers | json
```
Comment (alonsovb) [inline apps/nest-server/README.md:24]: can we remove the `json` part? it's not available everywhere

Comment (alonsovb) [reply apps/nest-server/README.md:24]: image

image

Comment (jorgeramirezamora) [reply apps/nest-server/README.md:24]: I added it as a dependency, you have to run a `npm i`

File: apps/nextjs/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+1. Start a local Presto instance using `docker-compose` by running the following command:
+
+   ```bash
+   npm run presto:up
+   ```
+
+2. After the Presto instance is up and running, you can start the Next.js application by running:
+
+   ```bash
+   npm run serve nextjs
```
Comment (aarong506) [inline apps/nextjs/README.md:18]: This does not work for me.

**Review by alonsovb — APPROVED** [review]

**Review by aarong506 — APPROVED** [review]
LGTM

**Review by yhwang — APPROVED** [review]
LGTM

**Pull request merged**